Tutorial Series/Introduction to Financial Python/Tutorial10 Multiple Linear Regression.html

```diff
@@ -194,7 +194,7 @@
     </tr>
 </table>
 
-As seen from the summary table, the p-values for Ebay, Walmart and Apple are 0.174, 0.330 and 0.068 respectively, so none of the them are significant at a 95% confidence.
+As seen from the summary table, the p-values for Ebay, Walmart and Apple are 0.174, 0.330 and 0.068 respectively, so none of them is significant at the 95% level.

```

Comment:
" so none of them is significant"
" so none of them are significant at **a** 95% confidence level"
plural, **are**

Reply:
Edited it in github.com/QuantConnect/Tutorials/pull/48
I was taught that "none of them" is singular. Just learnt that it can be singular or plural, depending on the emphasis needed:
www.onlinegrammar.com.au/top-10-grammar-myths-none-always-takes-a-singular-verb